src/openms_gui/CMakeLists.txt

```diff
@@ -55,7 +55,7 @@ IF (NOT Qt5Widgets_FOUND)
 	message(FATAL_ERROR "To find a custom Qt installation use: cmake <..more options..> -DCMAKE_PREFIX_PATH='<path_to_parent_folder_of_lib_folder_withAllQt5Libs>' <src-dir>")
 ENDIF()
 
-set(CMAKE_AUTOUIC ON)
+set(CMAKE_AUTOUIC OFF) # leave off since this is broken, see qt5_wrap_ui below
```

Comment:
is this broken in all versions of Qt5 or only the older ones?

Reply:
this is independent of Qt. Its a CMake issue

Reply:
ok but I thought that was fixed in later versions - might be wrong

Reply:
im running `cmake version 3.15.4` and its broken...